### PR TITLE
fix episode page navigation & allow for mini series

### PR DESCRIPTION
**Pull request by copperhead57**

### CODE REVIEW

File: trace.php [hunk 1]
```diff
@@ -410,42 +431,51 @@ function replace_javascript ($html, $js_page_type, $js_file_name)
...
-                                 $matches[1].'trace.php?videodburl=http://www.imdb.com'.$matches[2].$is_iframe.$matches[3],
+    if ($js_page_type != 'common')  // common needs no changes only saved to cache as required on root server
+    {
+        // for search bar interactive dropdown list
+        // process string - var c='<a href="'+a.url+"?ref_="+b+'" class="poster"';
+        $pattern = '/(var c=\'<a href=\"\'\+)(a\.url\+\"\?ref_=\"\+b\+\'\" class=\"poster"\')/';
+        preg_match($pattern, $js_file_data, $matches);
+//    echo "<br> list of imdb js files - "; var_dump($matches);
+        $js_file_data = preg_replace($pattern,
+                                     $matches[1].'"trace.php?'.$iframe_val.'&videodburl=http://www.imdb.com"+'.$matches[2],
```
Comment (kec2) [inline trace.php]: Should'n the URL be HTTPS?

File: trace.php [hunk 2]
```diff
@@ -410,42 +431,51 @@ function replace_javascript ($html, $js_page_type, $js_file_name)
...
+                                     $js_file_data);
+
+        // process string - class="moreResults" href="',g+=h+"/find?s=all&q="+A+"&ref_="+k+'sr_sm">',  
+        $pattern = '#(class=\"moreResults\" href=\")(\',g\+=h\+\"/find\?s=all&q=\"\+A\+\"&ref_=\"\+k\+\'sr_sm\">\',)#';
+        preg_match($pattern, $js_file_data, $matches);
+//    echo "<br> js file - find moreresults"; var_dump($matches);
+        $js_file_data = preg_replace($pattern,
+                                     $matches[1].'trace.php?'.$iframe_val.'&videodburl=http://www.imdb.com'.$matches[2],
```
Comment (kec2) [inline trace.php]: Should'n the URL be HTTPS?

File: trace.php [hunk 3]
```diff
@@ -410,42 +431,51 @@ function replace_javascript ($html, $js_page_type, $js_file_name)
...
+                                     $js_file_data);
+
+        // for season, year change drop down list on episode list
+        // process - if(d!==c){var e="/title/"
+        $pattern = '#(if\(d!==c\){var e=)(\"\/title\/\")#';
+        preg_match($pattern, $js_file_data, $matches);
+//    echo "<br> js file - find for season"; var_dump($matches);
+        $js_file_data = preg_replace($pattern,
+                                     $matches[1].'"trace.php?'.$iframe_val.'&videodburl=http://www.imdb.com"+'.$matches[2],
```
Comment (kec2) [inline trace.php]: Should'n the URL be HTTPS?

File: trace.php (hunk 1 above)
Comment (copperhead57) [reply trace.php]: will work with or without the 's'

File: trace.php (hunk 2 above)
Comment (copperhead57) [reply trace.php]: will work with or without the 's'

File: trace.php (hunk 3 above)
Comment (copperhead57) [reply trace.php]: will work with or without the 's'

Comment (kec2) [reply trace.php]: Cool.

File: trace.php (hunk 2 above)
Comment (kec2) [reply trace.php]: Cool.

File: trace.php (hunk 1 above)
Comment (kec2) [reply trace.php]: Cool.

**Comment by copperhead57** [comment]
closing to work on retrofitting kec2 changes

**Pull request closed without merging**